syntax = "proto3";

service UserService {
    rpc GetUserById (UserIdRequest) returns (UserResponse);
}

message UserIdRequest {
    int32 user_id = 1;
}

message UserResponse {
    int32 user_id = 1;
    string name = 2;
    string email = 3;
}